pennylane/ops/qubit/qchem_ops.py

```diff
@@ -1055,3 +1055,198 @@ def compute_decomposition(phi, wires):
     def adjoint(self):
         (phi,) = self.parameters
         return OrbitalRotation(-phi, wires=self.wires)
+
+
+class FermionicSWAP(Operation):
```

Comment:
Is there any relation between this gate and [FermionicSingleExcitation](https://docs.pennylane.ai/en/stable/code/api/pennylane.FermionicSingleExcitation.html)?

Reply:
I am not aware of any relation with [FermionicSingleExcitation](https://docs.pennylane.ai/en/stable/code/api/pennylane.FermionicSingleExcitation.html), but there does exist a relation to [iSWAP](https://docs.pennylane.ai/en/stable/code/api/pennylane.ISWAP.html) gate raised to power `phi`.